src/DistributedArrays.jl

```diff
@@ -10,6 +10,8 @@ export (.+), (.-), (.*), (./), (.%), (.<<), (.>>), div, mod, rem, (&), (|), ($)
 export DArray, SubDArray, SubOrDArray, @DArray
 export dzeros, dones, dfill, drand, drandn, distribute, localpart, localindexes, ppeval, samedist
 
+const registry=Dict{Tuple, Any}()
```

Comment:
I think we should re-use the per module metadata registry that the doc system uses.